llvm/lib/Target/AArch64/AArch64.td

```diff
@@ -179,3 +179,21 @@ def AArch64 : Target {
 //===----------------------------------------------------------------------===//
 
 include "AArch64PfmCounters.td"
+
+
+//===----------------------------------------------------------------------===//
+// GlobalISel patterns
+//===----------------------------------------------------------------------===//
+
+include "AArch64GlobalISelPatterns.td"
```

Comment:
Does this inflate the size of AArch64GenDAGISel.inc by including patterns that SelectionDAG doesn't need?

Reply:
```
> grep GenericInstruction llvm/include/llvm/Target/GenericOpcodes.td | wc -l
219
```
Yes, it will include patterns that SelectionDAG doesn't need/use.
There are 219 named opcodes. I have no numbers, but it will only be a small subset of them. There will be no G_FSINCOS patterns.